internal/jwt_authorization.go

```diff
@@ -40,24 +40,28 @@ func NewAdminJwtAuthorizationProvider(privateKey []byte) auth.AuthorizationProvi
 }
 
 func (j *JWTAuthProvider) GetAuthToken() ([]byte, error) {
-	claims := auth.JWTClaims{
-		Admin: true,
-		Iat:   time.Now().Unix(),
-		TTL:   60 * 10,
-	}
 	key, err := util.LoadRSAPrivateKey(j.PrivateKey)
 	if err != nil {
 		return nil, err
 	}
-	signer, err := jwt.NewSignerRS(jwt.RS256, key)
-	if err != nil {
-		return nil, err
+
+	ttl := int64(60 * 10)
```

Comment:
why it is hardcoded? Should it be configurable?

Reply:
No sure why this was hard-coded during the [initial implementation](https://github.com/uber-go/cadence-client/pull/1116) but my take is this: this ttl is for internal/self-signed tokens. There are no external calls to acquire a token, there's no caching involved, they are issued for every request, in other words, they are cheap and should be short lived.